app/views/govuk_component/breadcrumbs.raw.html.erb

```diff
@@ -3,10 +3,20 @@
 %>
 <div class="govuk-breadcrumbs">
   <ol>
-  <% breadcrumbs.each do |crumb| %>
+  <% breadcrumbs.each_with_index do |crumb, index| %>
     <li>
       <% if crumb[:url] %>
-        <a href="<%= crumb[:url] %>"><%= crumb[:title] %></a>
+        <%= link_to(
+          crumb[:title],
+          crumb[:url],
+          data: {
+            track_category: 'breadcrumbClicked',
+            track_action: index + 1,
+            track_label: crumb[:url],
+            track_dimension: crumb[:title],
```

Comment:
In the initial implementation I inferred most of the data via Javascript, but now that we are adding data attributes I thought it would be clearer to leave it there even if it looks a bit repetitive. This way I think it's clearer for someone in the future having a look. Does that make sense?